core/server/master/src/test/java/alluxio/master/file/FileSystemMasterSyncMetadataTest.java

```diff
@@ -281,9 +442,14 @@ InodeSyncStream.SyncStatus syncMetadata(RpcContext rpcContext, AlluxioURI path,
         @Nullable FileSystemMasterAuditContext auditContext,
         @Nullable Function<LockedInodePath, Inode> auditContextSrcInodeFunc,
         boolean isGetFileInfo) throws AccessControlException, InvalidPathException {
-      mSynced.set(true);
-      return super.syncMetadata(rpcContext, path, options, syncDescendantType, auditContext,
```

Comment:
the old logic here is not correct because the sync can be skipped but the mSynced is still set true